config/routes.rb

```diff
@@ -1,6 +1,28 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  devise_scope :user do
+    root to: "users/sessions#new"
+  end
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
```

Comment:
@Khanhpq-1999 

Registration thế có custom gì lại so với controller của devise đâu?

Reply:
registrations: 'users/registrations' em thêm vào để  Users::RegistrationsController xử lý sign_in thay vì
Devise::RegistrationsController xử lý. vì em muốn custom khi sign in xong thì sẽ redirect lại tới trang yêu cầu xác nhận email